backend/core/Extensions/DateExtensions.cs

```diff
@@ -33,7 +33,7 @@ public static int GetFiscalYear(this DateTime date)
         /// <returns></returns>
         public static string FiscalYear(this int fiscalYear)
         {
-            return $"{(fiscalYear - 1).ToString().Substring(2, 2)}/{(fiscalYear).ToString().Substring(2, 2)}";
+            return fiscalYear > 0 ? $"{(fiscalYear - 1).ToString().Substring(2, 2)}/{(fiscalYear).ToString().Substring(2, 2)}" : "";
```

Comment:
During debugging, it seems that one of the tests which instantiates a new project, passes in the fiscalYear as 0 which caused this to fail.

Reply:
You probably could have just updated the test to set the default value of "20/21".